lib/katello/tasks/clean_docker_meta_tags.rake

```diff
@@ -0,0 +1,8 @@
+require File.expand_path("../engine", File.dirname(__FILE__))
+
+namespace :katello do
+  desc "Reindex DockerMetaTags in response to Redmine #35233"
```

Comment:
Some downstream users may find this confusing. Doesn't seem like too much but at the end of the day it can actually generate support cases with inquiries about what does that mean.

In the future I think we need a better way to customize these types of strings for upstream and downstream users, like having defaults that are tailored for the experience of upstream users but can be overridden by drop-in files that can be included in downstream builds. That way an upstream user can see a link to a Redmine issue, or just explanation in upstream terminology "katello, content-proxy" and downstream builds could overwrite these with downstream terminology, knowledgebase links, etc.

Obviously that's not something we can solve right now. I could do a one-off overwrite when building an RPM for this but then someone else would have to do repeat it later if it goes into a z-stream release.

So it might be best to be a bit more general here for now.

```suggestion
  desc "Reindex DockerMetaTags"
```

Reply:
Actually, on second thought. Since this rake task would only run in the one-off scenario, this string should never be displayed during the migration switchover and would only be found by an user that went looking for this. So I think it's probably fine as it is?

Reply:
yeah they would need to run `foreman-rake --tasks` to see this.

Reply:
```
# foreman-rake --tasks
...
rake katello:clean_docker_meta_tags                                                  # Reindex DockerMetaTags in response to Redmine #35233
rake katello:clean_old_file_repos                                                    # Cleans up file repos that were moved to /pulp/isos/<org name>/
rake katello:clean_published_repo_directories                                        # Cleans up obsolete directories and estimates time to republish current directories
...
```